src/V1Meta.js

```diff
@@ -54,4 +54,10 @@ export default class V1Meta {
         const headers = createHeaderObj(this.authHeader);
         return Promise.resolve(this.postFn(url, null, headers));
     }
+
+    queryDefinition(assetType) {
+        const url = `${this.urls.meta()}/${assetType}`;
+        const headers = createHeaderObj(this.authHeader);
+        return Promise.resolve(this.postFn(url, null, headers));
```

Comment:
@JogoShugh When new-ing up an instance of V1Meta you provide it a post function (any lib or hand baked) so in this case you could give it a method that performs a HTTP GET, despite the sdk using `postFn` as a name.